libzmq recipy build fail

### Logs

```
[1m[90m[DEBUG][39m[0m:   	  CXX      src/src_libzmq_la-router.lo
[1m[90m[DEBUG][39m[0m:   	In file included from src/mtrie.cpp:32:
[1m[90m[DEBUG][39m[0m:   	./src/generic_mtrie_impl.hpp:52:46: error: ISO C++ requires the name after '::~' to be found in the same scope as the name before '::~' [-Werror,-Wdtor-name]
[1m[90m[DEBUG][39m[0m:   	template <typename T> zmq::generic_mtrie_t<T>::~generic_mtrie_t ()
[1m[90m[DEBUG][39m[0m:   	                      ~~~~~~~~~~~~~~~~~~~~~~~^~
[1m[90m[DEBUG][39m[0m:   	                                             ::generic_mtrie_t
[1m[90m[DEBUG][39m[0m:   	  CXX      src/src_libzmq_la-scatter.lo
[1m[90m[DEBUG][39m[0m:   	1 error generated.
```

--- a/pythonforandroid/recipes/libzmq/__init__.py
+++ b/pythonforandroid/recipes/libzmq/__init__.py
@@ -6,7 +6,7 @@ import sh
 
 
 class LibZMQRecipe(Recipe):
-    version = '4.3.2'
+    version = '4.3.4'
     url = 'https://github.com/zeromq/libzmq/releases/download/v{version}/zeromq-{version}.zip'
     depends = []
     built_libraries = {'libzmq.so': 'src/.libs'}
@@ -34,6 +34,7 @@ class LibZMQRecipe(Recipe):
                 '--prefix={}'.format(prefix),
                 '--with-libsodium=no',
                 '--disable-libunwind',
+                '--disable-Werror',
                 _env=env)
             shprint(sh.make, _env=env)
             shprint(sh.make, 'install', _env=env)

